Move section "Creating a NApp with UI" to the Web-UI documentation (#1167)
kytos/kytos: docs/developer/creating_a_napp.rst

- **ui**: Folder with components to be displayed in the Kytos UI.
- **ui/README.rst**: A file with a brief description of your NApp components.

> UI
> ---

Reviewer: ```suggestion
User Interface
--------------
```

> You can create a NApp with UI, using our custom `Vue.js` and HTML components. 
> To learn more visit our 

Reviewer: ```suggestion
You can create a web UI for your NApp using our custom `Vue.js` and 
HTML components. To learn more, visit our
```

> `Web Interface documentation <https://docs.kytos.io/developer/web-ui/>` _

Reviewer: Is there a specific need to create absolute links? If not, it's better to use relative:
```suggestion
`Web Interface documentation </developer/web-ui/>` _
```
